### DISCUSSION TITLE
Mi cortadora láser falcon 5w no quema

### DISCUSSION THREAD

**Discussion by ansay27** [arkypita/LaserGRBL#2181] [Q&A] [answered]
buenas a tod@s, soy nueva en este tema del corte láser, compramos la máquina y solo la ocupamos una sola vez, grabando una imagen, pero ahora, no lo realiza, se conecta con el programa de grbl, se mueve el sensor y todo, pero no quema nada, alguien que me ayude con eso por favor, gracias. saludos

**Comment by Tribo-2dot0** [comment]
O feixe de laser acende quando supostamente estaria gravando?

**Reply by ansay27** [reply, marked as answer]
Si enciende la luz roja

**Comment by Tribo-2dot0** [comment]
Desconecte o cabo do laser e reconecte-o, pode ser um mal contato. Verifique também se os pequenos fios que compõem o cabo estão firmemente conectados ao terminal (plug).
